fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -564,11 +574,10 @@ module.exports = function(config, DB) {
 
             // Attempt to fetch a deleted session token
             return db.sessionToken(SESSION_TOKEN_ID)
-          })
-          .then(function(token) {
-            assert(false, 'Session Token should no longer exist')
-          }, function(err) {
-            // 'Session Token deleted successfully'
+              .then(
+                () => assert(false, 'Session Token should no longer exist'),
+                () => assert('Session token was deleted successfully')
```

Comment:
The generic catch-all handler that was here was actually preventing these tests from failing when I had a problem in my implementation. I moved it in with the call it's meant for so that other errors can propagate and correctly fail the suite.